src/java.base/share/classes/sun/net/ftp/impl/FtpClient.java

```diff
@@ -921,6 +918,7 @@ private void tryConnect(InetSocketAddress dest, int timeout) throws IOException
         in = new BufferedInputStream(server.getInputStream());
     }
 
+    @SuppressWarnings("removal")
```

Comment:
Could the scope be further reduced by applying it at line 926 in this method - at the cost of creating a temporary variable? The code could probably be further simplified by using a lambda...

```
            PrivilegedAction<Socket> pa = () -> new Socket(proxy);
            @SuppressWarnings("removal")
            var ps = AccessController.doPrivileged(pa);
            s = ps;
```